src/SCRIPTS/BF/ui.lua

```diff
@@ -60,6 +60,7 @@ end
 
 local function invalidatePages()
     Page = nil
+    dataInitialised = false
```

Comment:
This shouldn't be invalidated - no point in re-reading data over and over again, especially as bandwidth is already scarce.